{"uuid":"fb0fb089-727b-4b3f-b36a-27b4a12daafe","historyId":"56c711282cea76c0cfdc11fb4ee2e974","testCaseId":"[engine:junit-jupiter]/[class:wikipedia.test.ExceptionTest]/[method:exception()]","testCaseName":"exception()","fullName":"wikipedia.test.ExceptionTest.exception","labels":[{"name":"junit.platform.uniqueid","value":"[engine:junit-jupiter]/[class:wikipedia.test.ExceptionTest]/[method:exception()]"},{"name":"host","value":"VNE-NN-22-230"},{"name":"thread","value":"33436@VNE-NN-22-230.main(1)"},{"name":"framework","value":"junit-platform"},{"name":"language","value":"java"},{"name":"package","value":"wikipedia.test.ExceptionTest"},{"name":"testClass","value":"wikipedia.test.ExceptionTest"},{"name":"testMethod","value":"exception"},{"name":"suite","value":"wikipedia.test.ExceptionTest"}],"links":[],"name":"exception()","status":"passed","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1745503528164,"stop":1745503528272}